### DISCUSSION TITLE
release_notes.json is not working on App Store Connect for Arabic

### DISCUSSION THREAD

**Discussion by BWhiteApps** [codemagic-ci-cd/codemagic-docs#2708] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

I have a `release_notes.json` file that looks like this:
```
[
  {
    "language": "en-US",
    "text": "Some release notes here"
  },
  {
    "language": "ar",
    "text": "Some release notes here"
  },
  {
    "language": "ru",
    "text": "Some release notes here"
  },
]
```

All language codes seem to work except for Arabic which should be "ar". This forces me to manually enter them each time I push to the store. Is there a different localization code I should be using?

### Expected results

Release notes get added for Arabic

### Actual results

Arabic is left out while other languages are added.

**Comment by mikhail-tokarev** [comment]
Hey

Try "ar-SA" according to Apple docs

https://developer.apple.com/documentation/appstoreconnectapi/app_store/app_metadata/app_info_localizations/managing_metadata_in_your_app_by_using_locale_shortcodes

**Comment by himesh-cm** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
